### PR TITLE
fix: [DEL-5540]: add new command for helm delegate

### PR DESCRIPTION
**Pull request by swarajcheguri-harness**
### Summary
Add a new command to Helm Delegate commands displayed on UI

image

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by rameshx — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

File: src/modules/30-delegates/strings/strings.en.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ successVerification:
   installHelmChartLabel: 'To install the helm chart use the below command'
   installHelmChartCmd: '$ helm install  --namespace <namespace> --create-namespace harness-ng/harness-delegate-ng -f harness-delegate-values.yaml'
   checkingForHeartbeat: Checking for heartbeat
+  updateHelmRepoLabel: 'Update Harness helm repo'
+  helmRepoUpdate: helm repo update harness-ng
```
Comment (rameshx) [inline src/modules/30-delegates/strings/strings.en.yaml:41]: This looks like a command, so it shouldn't be translated.

**Review by rameshx — CHANGES_REQUESTED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/30-delegates/strings/strings.en.yaml (hunk 1 above)
Comment (swarajcheguri-harness) [reply src/modules/30-delegates/strings/strings.en.yaml:41]: it follows other commands  like [this](https://github.com/harness/harness-core-ui/pull/13469/files#diff-fa8066060ac1a347c11932c8fe87f22304af4683c0f0c0e92de2f8dab47bee29R38)
So should we change all other commands too ? @rameshx  ?

Comment (rameshx) [reply src/modules/30-delegates/strings/strings.en.yaml:41]: As discussed, commands can be excluded from adding to strings yaml

**Review by rameshx — APPROVED** [review]

File: src/modules/30-delegates/strings/strings.en.yaml (hunk 1 above)
Comment (swarajcheguri-harness) [reply src/modules/30-delegates/strings/strings.en.yaml:41]: Created a task to do the same https://harness.atlassian.net/browse/DEL-5817

**Pull request merged**